### PR TITLE
Small patches

### PR DESCRIPTION
**Pull request by lucernae**
@Gustry some small patches. It doesn't have to be merged. Just FYI to know what I did to hack this.

- Add patch to make sure style with no name will always have `default`
- Fix set_default_style call to include parameters

### CODE REVIEW

File: filters/style_manager.py [hunk 1]
```diff
@@ -109,6 +109,12 @@ def responseComplete(self):
                 qgis_layer = qgis_layer[0]
 
             # NAME
+            # Check default name should be `default` to follow GetCapabilities
+            # response
+            style_manager = qgis_layer.styleManager()
+            style_manager.renameStyle('', 'default')
+            project.write()
```
Comment (lucernae) [inline filters/style_manager.py:116]: What I did to rename the style from empty to `default`.
Just a temporary hack for me until we figured out what the problem.

File: filters/style_manager.py [hunk 2]
```diff
@@ -123,7 +129,8 @@ def responseComplete(self):
             elif service == 'GetStyle':
                 return self.get_style(name, qgis_layer, request)
             elif service == 'SetDefaultStyle':
-                return self.set_default_style()
+                return self.set_default_style(
+                    name, project, project_path, qgis_layer, request)
```
Comment (lucernae) [inline filters/style_manager.py:133]: Perhaps you forgot this one? Just to remind you for the next PR.
The functionality itself is working.

**Review by lucernae — COMMENTED** [review]
I tried these requests (GetStyle, AddStyle, RemoveStyle, SetDefaultStyle) and it is working with valid name (I'm using postman).

Also a general question, do I need to use specific HTTP Request method? I'm using GET request for all of these (it is working).

**Comment by Gustry** [comment]
> Also a general question, do I need to use specific HTTP Request method? I'm using GET request for all of these (it is working).

Yes, just use a basic GET HTTP. In QGIS 3, it's easier to set the HTTP code. Maybe it's easier also to work with DELETE, PUT etc

**Comment by Gustry** [comment]
Thanks, looks good!

**Pull request merged**